### ISSUE TITLE
I'm slow and need help

### ISSUE THREAD

**Issue by TheRichRicky** [PWhiddy/PokemonRedExperiments#83]
I have no clue what im doing, i'm trying to understand the code and instructions and need help.

Can someone please add a video or dumb it down for me to install this program

Please Help
TheRichRicky

**Comment by TheRichRicky** [comment]
> https://youtu.be/DcYLT37ImBY
> 
> Here's the video. Watch it until the end with the transcript turned on
> 
> Issue solved.

What Transcript? you mean the INSTRUCTIONS? (im not a coder and dont know the lingo)

**Comment by akraus53** [comment]
Hey @TheRichRicky, welcome to the world of Open Source Code.

I highly encourage you to start learning programming if that's something you're interested in, but this topic is really on a high level - too high for someone who's not a coder. I really don't mean to sound elitist, but using this to get into coding is like buying a plane ticket to Tokio because you're interested in Japan. One day, sure, but start slow. If you want to get an easy way into programming, look at the content of Daniel Shiffman on YouTube (TheCodingTrain), he just remade his Introduction to Programming series which is amazing and once helped me find my way into programming. Starting with a topic like this will be frustrating as heck. 

If you do choose to continue to try to get this to work, I recommend asking ChatGPT with questions about shasum and what errors in your console mean. 

Good luck and happy programming!!
